src/styles/sass/pages/_home.scss

```diff
@@ -0,0 +1,316 @@
+                flex-basis: calc(50% - 1rem);
+            }
+        }        
+
+        .custom-label {
+            display: none;
+        }
+
+        .custom-input {
+            width: calc(33% - 1rem);
+            margin: 0 0 0 0;
+            border: none;
+            border-radius: 5px;
+            padding: 0 0.5rem 0 0.5rem;
+            line-height: 0;
+            height: 57.6px;
+            color: $clr-darkGreen;
+            text-align: right;
+
+            &::placeholder {
+                text-align: center;
+                color: $clr-lightGrey;
+            }
+
+            &:focus,
+            &:active {
+                outline: none;
+                border: 2px solid $clr-green;
+            }
+
+            @media screen and (max-width: 1200px) {
+                width: calc(50% - 1rem);
+            }
+        }
+
+        input[type="radio"]:checked + label {
+            background: $clr-green;
+            color: $clr-darkGreen;
+        }
+    }
+}
+
+.bill-display-container {
+    width: 50%;
+    border-radius: 25px;
+    padding: 2rem;
+    background-color: $clr-darkGreen;
+
+    @media screen and (max-width: 1200px) {
+        width: 100%;
+    }
+
+    .gross-bill-per-person, .tip-per-person, .total-bill-per-person {
+        display: flex;
+        margin: 0 0 1rem 0;
+    }
+
+    .display-label-container {
+        display: flex;
+        flex-direction: column;
+        margin: 0 0 1.5rem 0;
+        width: 50%;
+        color: $clr-white;
+
+        .per-person-label {
+            @include fontSize(16);
+            margin: 0.25rem 0 0 0;
+            color: $clr-grey;
+        }
+    }
+
+    .amount-display {
+        @include fontSize(46);
+        width: 50%;
+        min-width: 200px;
+        color: $clr-green;
+        text-align: center;
+    }
+
+    .button-container {
+        display: flex;
+        flex-direction: column;
+    }
+
+    button {
+        margin: 1rem 0 1rem 0;
+        border: none;
+        border-radius: 10px;
+        background: $clr-green;
+        color: $clr-darkGreen;
+        height: 50px;
+
+        &:hover,
+        &:focus {
+            cursor: pointer;
+            color: $clr-darkGreen;
+            background: $clr-lighterGreen;
+        }
+    }
+}
+
+/* Bill Transaction History Styles */
+.bill-history-section {
+    width: 100%;
+
+    h2 {
+        text-align: center;
+        color: $clr-grey;
+    }
+
+    .transaction-table {
+        table-layout: fixed;
+        max-width: 100%;
+
+        tr {
+            @include fontSize(16);
+        }
+
+        th {
+            @include fontSize(20);
+            width: 200px;
+            height: 50px;
+        }
+
+        td {
+            width: 200px;
+            height: 50px;
+            text-align: center;
+            color: $clr-grey;
+        }
+
+        @media screen and (max-width: 480px) {
+            width: 100%;
+        }
+    }
+
+    @media screen and (max-width: 480px) {
```

Comment:
I miss the media query for some intermediary devices. Maybe you could implement it for the next split.

Reply:
Sounds good, noted.